packages/@aws-cdk/aws-route53/test/test.hosted-zone-provider.ts

```diff
@@ -15,8 +15,9 @@ export = {
       const missing = SynthUtils.synthesize(stack).assembly.manifest.missing!;
       test.ok(missing && missing.length === 1);
 
+      const fakeZoneId = '11111111111111';
       const fakeZone = {
```

Comment:
Please add a test which verifies that if users pass a hosted zone without the “/hostedzone” prefix, we still get the desired effect.

Reply:
Done, please see https://github.com/aws/aws-cdk/pull/5230/commits/667f65ff312af2966c383e779bff4d74d3e95324